test/cell_test.rb

```diff
@@ -0,0 +1,113 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/ship'
+require './lib/cell'
+
+class CellTest < Minitest::Test
+
+  def setup
+    skip
+    @cell = Cell.new("B4")
+  end
+
+  def test_it_exists
+    skip
+    assert_instance_of Cell, @cell
+  end
+
+  def test_it_has_coordinates
+    skip
+    assert_equal "B4", @cell.coordinate
+  end
+
+  def test_it_does_not_have_a_ship
+    skip
+    assert_equal nil, @cell.ship
+  end
+
+  def test_it_is_empty
+    skip
+    assert @cell.empty?
+  end
+
+  def test_it_has_a_new_ship
+    skip
+    assert_equal @cruiser, @cell.place_ship
+  end
+
+  def test_it_is_not_empty
+    skip
+    refute @cell.empty?
+  end
+
+  def test_it_has_not_been_fired_upon
+    skip
+    refute @cell.fired_upon?
+  end
+
+  def test_when_hit_loses_1_health
+    @cruiser.hit
```

Comment:
@amymoller 
While this is technically what will happen, it happens _after_ the cell gets fired upon. So in the interaction pattern it wants us to call `@cell.fire_upon` first, which will in effect cause the ship to get `.hit`, but the way we are showing this is with your next line: `assert_equal 2, @cell.ship.health` which is correct! 

(not by calling `@cruiser.hit` because we are testing Cell methods in this file instead of Ship methods. `.hit` only works on Ship objects!)

Reply:
I totally get this more now! Yea I wasn't 100% sure if it was kosher to call on @cruiser.hit haha.  Thanks! Looking over all this with fresher eyes definitely helps :)